iam-login-service/src/main/java/it/infn/mw/iam/core/error/InvalidResourceError.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Istituto Nazionale di Fisica Nucleare (INFN). 2016-2021
```

Comment:
At least for new files, let's use the right year(s). In this case it would just be 2024.

Reply:
Ok, maybe we can add a PR which updates all the licenses, otherwise the workflow would fail here https://github.com/indigo-iam/iam/blob/master/.github/workflows/maven.yml#L27.